Public/Src/App/Bxl/BuildXLApp.cs

```diff
@@ -1790,6 +1795,13 @@ internal bool OnConsoleCancelEvent(bool isTermination)
 
                     if (!OperatingSystemHelper.IsUnixOS)
                     {
+                        string[] filesToAttach = m_configuration != null

```

Comment:
>m_configuration != null [](start = 49, length = 23)

i know that you only moved that code, but I do not see how `m_configuration` can be null, i.e., I think this check is not necessary

Reply:
It can in fact be null due to the wacky configuration swapping that happens during Engine.Run(). But your point is still valid since this block no longer requires it to be non-null since I am working off of loggers instead of m_configuration.Logging

---
In reply to: [316293989](https://github.com/microsoft/BuildXL/pull/754#discussion_r316293989) [](ancestors = 316293989)